yarn-project/end-to-end/src/e2e_dapp_subscription.test.ts

```diff
@@ -52,13 +53,22 @@ describe('e2e_dapp_subscription', () => {
   const PRIVATELY_MINTED_BANANAS = 600n;
 
   const FEE_AMOUNT = 1n;
-  const REFUND = 2n; // intentionally overpay the gas fee. This is the expected refund.
+  const REFUND = 29n; // intentionally overpay the gas fee. This is the expected refund.
   const MAX_FEE = FEE_AMOUNT + REFUND;
 
+  const GAS_SETTINGS = GasSettings.new(
```

Comment:
Nit: its a bummer that someone needs to look at the variable names of `new` to know what the args are. might be nice to allow a full object to be passed in the `from` style like
```
{
  da: {gasLimit:5 ...},
  l1: ...
}
```